src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -179,5 +179,10 @@ internal class CommonLocalizableStrings
         public const string SpecifiedNameExists = "Specified name {0} already exists. Please specify a different name.";
         public const string SpecifiedAliasExists = "Specified alias {0} already exists. Please specify a different alias.";
         public const string MandatoryParameterMissing = "Mandatory parameter {0} missing for template {1}. ";
+
+        /// compatibility
+        public const string ProjectNotCompatibleWithFramework = "Project `{0}` is not compatible with `{1}`.";
```

Comment:
@blackdwarf Can you validate these strings? Not sure if this is enough information for the user to know what to do with it. I mean, not compatible in what sense? What are the dimensions that these projects are being compared against?